client/src/components/Feed/Post.js

```diff
@@ -50,80 +62,137 @@ const Post = ({
   // mock API to test functionality
   /* to be removed after full integration with user api */
   const [shared, setShared] = useState(false);
-  const [comment, setComment] = useState("");
   const [fakeShares, setFakeShares] = useState(0);
 
   const handlePostDelete = () => {
     setModalVisibility(!modalVisibility);
   };
 
-  const loadComments = useCallback(async () => {
+  const loadMoreComments = async () => {
+    setLoading(true)
+    if (firstLoad) {
+      page.current = page.current + 2
+      setFirstLoad(false)
+    } else {
+      page.current = page.current + 1
+    }
+    let response;
+    const postId = post._id;
+
+    const skip = 0;
+    const endPoint = `/api/posts/${postId}/comments?limit=${limit.current * page.current}&skip=${skip}`;
+
+    try {
+      response = await axios.get(endPoint);
+    } catch (error) {
+      console.log({ error });
```

Comment:
need to display or handle an error appropriately?

Reply:
Thanks! This is super helpful!